### ISSUE TITLE
Unchecking Modifier's not working?

### ISSUE THREAD

**Issue by Nightmare-246** [Mustard2/MustardSimplify#26]
As the title says, the Modifiers I select to not be simplified isn't working. It just gets simplified anyway despite the fact that I uncheck it. I've had this issue for a while now and idk if it's a bug or if I did something before that's causing this problem. The unchecking works for Shape keys and Objects, but not modifiers.

https://github.com/Mustard2/MustardSimplify/assets/172527043/15270e9c-6575-4c3d-865c-b14dc3746b8b

**Comment by cl3m3c7** [comment]
it actually works!
if you look closely, the modifier is not disabled because of the 'modifiers' option, (viewport icon in the modifier is still enabled!), but because of the 'blender simplify' option!

**Comment by Mustard2** [comment]
Yes exactly, thank you @cl3m3c7 ! This might be confusing, maybe I can add a warning or something.

Essentially, Blender Simplify is reducing the level of subdiv to 0, effectively deactivating it. But it's not the Modifier option, since as you can see from the viewport/render show options (the blue buttons on the right of the modifier name) are not disabled.

**Comment by Mustard2** [comment]
Closing this as it is not a bug.

If you encounter another issue, please do not hesitate to open a new Issue.
